BeatTogether.DedicatedServer.Kernel/Enums/SongSelectionMode.cs

```diff
@@ -4,7 +4,7 @@ public enum SongSelectionMode
     {
         Vote = 0,
         Random = 1,
-        OwnerPicks = 2,
+        ManagerPicks = 2,
```

Comment:
Changed to ManagerPicks to align with the instance manager being referred to a manager not owner